src/dev-spaces-preview/azext_dev_spaces_preview/custom.py

```diff
@@ -37,30 +37,17 @@ def ads_use_dev_spaces(cluster_name, resource_group_name, update=False, space_na
 
     azds_cli = _install_dev_spaces_cli(update)
 
-    from subprocess import PIPE
-    retCode = subprocess.call(
-        [azds_cli, 'controller', 'select', '--name', cluster_name, '--resource-group', resource_group_name],
-        stderr=PIPE)
-    if retCode != 0:
-        retCode = subprocess.call(
-            [azds_cli, 'controller', 'create', '--target-name', cluster_name, '--target-resource-group',
-             resource_group_name, '--name', cluster_name, '--resource-group', resource_group_name],
-            universal_newlines=True)
-        if retCode != 0:
-            return
-
-    if space_name is None:
-        space_name = 'default'
-
-    retCode = subprocess.call(
-        [azds_cli, 'space', 'select', '--name', space_name], stderr=PIPE)
-    if retCode == 0:
-        return
-
-    create_space_arguments = [azds_cli, 'space', 'create', '--name', space_name]
+    use_command_arguments = [azds_cli, 'use', '--name', cluster_name,
+                             '--resource-group', resource_group_name]
+
+    if space_name is not None:
+        use_command_arguments.append('--space')
+        use_command_arguments.append(space_name)
+
     if prompt:
```

Comment:
prompt is more of a "**don't prompt** for input" parameter right? might want to change the naming to something like `yes`.